client/extensions/woocommerce/state/sites/settings/stripe-connect-account/test/reducer.js

```diff
@@ -343,4 +347,176 @@ describe( 'reducer', () => {
 			expect( newState[ 456 ].settings.stripeConnectAccount.isRequesting ).to.eql( true );
 		} );
 	} );
+
+	// TODO
```

Comment:
Still need these `TODO`s?  Tests look OK to me.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/19244/commits/2d81a7a15a8b02918fd6b2ef39b9a54500e0b52c